static/css/impala/users.less

```diff
@@ -50,7 +50,7 @@
 .html-rtl .prettyform {
     .avatar,
     li.profile-photo a.delete {
-        margin-right: 137px;
+        margin-right: 0px;
```

Comment:
For a zero value we leave off the unit. So in this case it's preferred to use `0` instead of `0px`